index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Abdul Ali Noori's portfolio" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="top-navigation">
+            <a href="#headline" class="welcome-logo">Welcome</a>
+            <img src="images/Icons/Icon-Menu.svg" alt="GitHub">
```

Comment:
- [ ] Please as the last reviewer said delete this weird border around the page as it doesn't match the design.
![image](https://user-images.githubusercontent.com/74744787/212492945-3e7bb422-2194-4965-a12c-9f7e43000f3f.png)